ssh/data/run.sh

```diff
@@ -47,8 +47,14 @@ touch /data/.bash_history
 chmod 600 /data/.bash_history
 ln -s -f /data/.bash_history /root/.bash_history
 
-# Store token for Hass.io API
-echo "export HASSIO_TOKEN=${HASSIO_TOKEN}" >> /root/.bash_profile
+# Persist .bash_profile by redirecting .bash_profile to /data
+if bashio::fs.file_exists /data/.bash_profile; then
+  sed -i "s/export HASSIO_TOKEN=.*/export HASSIO_TOKEN=${HASSIO_TOKEN}/" /data/.bash_profile
+else
+  echo "export HASSIO_TOKEN=${HASSIO_TOKEN}" > /root/.bash_profile
+  chmod 600 /data/.bash_profile
```

Comment:
Let's move this out of the if statement.
That way, we can guarantee it is always correct. For example, when a user changes it or on the restore of a backup.

Reply:
Good point